apps/mail/app/(auth)/login/login-client.tsx

```
@@ -1,9 +1,11 @@
 'use client';
 
+
 import { useEffect, type ReactNode, useState, Suspense } from 'react';
 import { Google } from '@/components/icons/icons';
 import { type EnvVarInfo } from '@/lib/auth-providers';
 import { signIn, useSession } from '@/lib/auth-client';
+import { Google } from '@/components/icons/icons';
```

Biome (1.9.4)

[error] Shouldn't redeclare 'Google'. Consider to delete it or rename it.

'Google' is defined here:


(lint/suspicious/noRedeclare)